net-misc/seafile/seafile-7.0.5.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python{2_7,3_6})
+
+WANT_AUTOMAKE=1.16
+
+inherit autotools python-single-r1 vala
+
+DESCRIPTION="File syncing and sharing software with file encryption and group sharing"
+HOMEPAGE="https://github.com/haiwen/seafile/ http://www.seafile.com/"
```

Comment:
I'd swap the places of these homepages, and put Github as the secondary homepage here.